modules/mfa/public/send-recovery-mfa.php

```diff
@@ -63,7 +63,7 @@
 
 $t = new Template($globalConfig, 'mfa:send-recovery-mfa');
 $t->data['recovery_contacts_by_name'] = $recoveryContactsForView;
-$t->data['manager_email'] = $state['managerEmail'];
+$t->data['masked_manager_email'] = $state['managerEmail'];
```

Comment:
Is that value masked in the first place?

Reply:
In Mfa.php:

```php
$state['managerEmail'] = self::getMaskedManagerEmail($state);
```

Maybe the state array key needs to be renamed also? (I'm fine if it's left as-is.)

Reply:
I would like to rename it in the state array as well, but that was a slightly more in-depth change than I wanted to do right now.